src/hooks/types/Publisher.ts

```diff
@@ -0,0 +1,9 @@
+export type PublisherState = "ready" | "connecting" | "streaming";
+
+export interface Publisher {
```

Comment:
We don't need another folder for types, please move this into usePublisher hook

Reply:
And gone